tests/unit/ViolationUtilsTest.ts

```diff
@@ -129,9 +131,12 @@ describe('getViolationsOnyxData', () => {
                     name: 'Meals',
                     required: true,
                     tags: {
-                        Lunch: {name: 'Lunch', enabled: true},
-                        Dinner: {name: 'Dinner', enabled: true},
+                        // eslint-disable-next-line @typescript-eslint/naming-convention
+                        Lunch: {name: 'Lunch', enabled: true, 'GL Code': ''},
```

Comment:
If we keep the suppressions, let's add a whitespace before the comment
```suggestion
                        Lunch: {name: 'Lunch', enabled: true, 'GL Code': ''},
                        
```